# Network status: wifi signallevel and ethernet connection status

float32     wifi_signallevel
string	    wifi_link_quality
bool        ethernet_connected
